package.json

```diff
@@ -10,8 +10,11 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "classnames": "^2.3.2",
```

Comment:
classnames are not needed in presence of clsx, you can remove them

Reply:
done